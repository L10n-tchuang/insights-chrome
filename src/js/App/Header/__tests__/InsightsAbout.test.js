import React from 'react';
// import { BrowserRouter as Router } from 'react-router-dom';
import configureMockStore from 'redux-mock-store';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import InsightsAbout from '../InsightsAbout';
import fetch from 'node-fetch';

const mockStore = configureMockStore();

describe('InsightsAbout component', () => {

    let store;

    beforeEach(() => {
        const initialState = {
            chrome: { 
                user: { 
                    identity: { 
                        user: {}
                    } 
                }, 
                appId: 'fakeAppId',
                globalNav: '',
                activeApp: 'fakeAppId'
            }
        };

        store = mockStore(initialState);
    });

    it('should render correctly', () => {
        const wrapper = shallow(
            <InsightsAbout store={ store }/>
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});