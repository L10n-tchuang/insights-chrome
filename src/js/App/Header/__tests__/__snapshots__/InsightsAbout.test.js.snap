// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsightsAbout component should render correctly 1`] = `
<ContextProvider
  value={null}
>
  <InsightsAbout
    activeApp="fakeAppId"
    appId="fakeAppId"
    dispatch={[Function]}
    globalNav=""
    store={
      Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      }
    }
    user={Object {}}
  />
</ContextProvider>
`;
